fe1-web/components/ConfirmModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  View, Modal, Text,
+} from 'react-native';
+import STRINGS from 'res/strings';
+import styles from 'styles/stylesheets/modal';
+import WideButtonView from './WideButtonView';
+
+/**
+ * A modal used to ask for the confirmation or cancellation of the user.
+ */
+
+const ConfirmModal = (props: IPropTypes) => {
+  const { visibility } = props;
+  const { setVisibility } = props;
+  const { title } = props;
+  const { description } = props;
+  const { buttonConfirmText } = props;
+  const { buttonCancelText } = props;
+  const { onConfirmPress } = props;
+
+  return (
+    <Modal
+      visible={visibility}
+      transparent
+    >
+      <View style={styles.modalView}>
+        <View style={styles.titleView}>
+          <Text style={styles.modalTitle}>{title}</Text>
+        </View>
+        <Text style={styles.modalDescription}>{description}</Text>
+        <View style={styles.buttonView}>
+          <WideButtonView
+            title={buttonConfirmText}
+            onPress={() => onConfirmPress()}
+          />
+          <WideButtonView
+            title={buttonCancelText}
+            onPress={() => setVisibility(!visibility)}
+          />
+        </View>
+      </View>
+    </Modal>
+  );
+};
+
+const propTypes = {
+  visibility: PropTypes.bool.isRequired,
+  setVisibility: PropTypes.func.isRequired,
+  title: PropTypes.string.isRequired,
+  description: PropTypes.string.isRequired,
+  buttonCancelText: PropTypes.string,
+  buttonConfirmText: PropTypes.string,
+  onConfirmPress: PropTypes.func.isRequired,
```

Comment:
I'm going to be very annoying here (same thing applies to the other modal), but it seems to me that the API is mixing different level of abstractions.

On one hand we have `onConfirmPress`, which is indicating the result of the modal. It seems like an obvious abstraction.
On the other hand, we have `setVisibility`, which seems like a concern for the caller of the modal.

I'd really consider having just `onConfirm` and `onCancel`, and let the callers handle the visibility (ideally in *both* cases, not just on cancel).

I understand this would lead to a tiny bit of code repetition, but it should also lead to a clearer separation of concerns and easier re-usability for the modals.